### ISSUE TITLE
I wonder what "pt" and "pa" stand for respectively

### ISSUE THREAD

**Issue by wjl2244** [ZikangZhou/QCNet#26]
hellow! I wonder what "pt" and "pl" stand for in code respectively? is "point" and "polygon"?

**Comment by wjl2244** [comment]
is "polygon" and "point"?

**Comment by ZikangZhou** [comment]
Yes, you're right. The map encoder is similar to VectorNet, which learns map features in a hierarchical way to reduce the number of map nodes.
